standard.md

```diff
@@ -47,8 +48,45 @@ We also believe that creating momentum for Saas models in the Ethereum space is
 
 ## Specification
 
-TODO
+### Public Methods
+
+#### isValidSubscription
+Used to check whether a given subscription ID represents a valid subscription.
+
+* @param  \_subscriptionId The subscription ID to check
+* @return a boolean to indicate whether the subscription exists and is valid
+
+```
+function isValidSubscription(bytes32 _subscriptionId)
```

Comment:
What is the purpose of having this as a separate function instead of just a part of a tupple that getSubscription() returns?